cmd/cluster/instance_migrate.go

```diff
@@ -58,6 +74,39 @@ func init() {
 	replaceVoyagerCmd.PersistentFlags().StringP("out", "o", "", "The name of the file in which the AstarteVoyagerIngress custom resource will be saved.")
 
 	MigrateCmd.AddCommand(replaceVoyagerCmd)
+	MigrateCmd.AddCommand(updateStorageVersionCmd)
+}
+
+func updateStorageVersionCmdF(command *cobra.Command, args []string) error {
+	crds := []string{"astartes.api.astarte-platform.org", "astartevoyageringresses.api.astarte-platform.org", "flows.api.astarte-platform.org"}
+
+	crdsWithStoredVersions := map[string][]string{}
+	//check that required CRDs exist and save their current storedVersions, in case you need to restore them.
+	for _, v := range crds {
+		crd, err := kubernetesAPIExtensionsClient.ApiextensionsV1().CustomResourceDefinitions().Get(context.Background(),
```

Comment:
Can we switch to the form `if crd, err := <...>; err != nil {`?

Reply:
Here is ok to do as you say, because `crd` will be used in a one-line `else` branch